radiation/FatesTwoStreamInterfaceMod.F90

```diff
@@ -0,0 +1,533 @@
+Module FatesTwoStreamInterfaceMod
```

Comment:
Per our discussion, reword `Interface` to `Utils` (or other word) to avoid confusing this as being related to other API interface modules.